models/artifact_helpers_test.go

```diff
@@ -158,18 +214,6 @@ func TestArtifactValidation(t *testing.T) {
 		params   models.CreateArtifactParams
 		errorMsg string
 	}{
-		{
```

Comment:
Why this test is deleted?

Reply:
this is a duplicate of https://github.com/percona/pmm-managed/pull/748/files#diff-5a898b5b01f7ff28425211fc14a0224a0c34ecb231119a14a8066316aa836767R93